bayesian.go

```diff
@@ -254,6 +324,38 @@ func (c *Classifier) Learn(document []string, which Class) {
 	c.learned++
 }
 
+// Here we use all the TF samples for the class and convert
+// them to TF-IDF https://en.wikipedia.org/wiki/Tf%E2%80%93idf
+// once we have finished learning all the classes and have the totals.
+func (c *Classifier) ConvertTermsFreqToTfIdf() {
+
+	if c.DidConvertTfIdf {
+		panic("TfIdf-Mode:Cumulative counts - can only call this once. Reset and relearn.")
```

Comment:
Last nitpicks, I promise! :)

"Cannot call ConvertTermsFreqToTfIdf more than once. Reset and relearn to reconvert."